zingolib/src/lightclient.rs

```diff
@@ -163,6 +162,8 @@ pub struct LightClient {
     bsync_data: Arc<RwLock<BlazeSyncData>>,
     interrupt_sync: Arc<RwLock<bool>>,
 }
+
+/// We used to have an impl LightClient that was gated on test code, should we reintroduce that?
```

Comment:
this PR does implement this change right? do we need the comment now?

Reply:
Good catch!

Reply:
Fixed.